individuals.md

```diff
@@ -7,3 +7,4 @@ you.
 | Name | Employer | Lawyer? | Working Languages | Keywords |
 |---|---|---|---|---|
 | [James Shubin](https://purpleidea.com/) | self-employed | [ ] | English, French | [hacker, legal nerd, list founder ](individuals/james-shubin.md) |
+| [Carlo Piana](https://www2.array.eu/info/#piana/) | Array | [x] | English, Italian | [IT lawyer, avvocato ](individuals/carlo-piana.md) |
```

Comment:
You could add a link around "Array" too if you'd like, but it's optional.

Reply:
Also, it's not alphabetical, but I didn't write a test for that yet :(

Reply:
That's fixed. 
I am not at all familiar with the entire process, I am afraid to screw everything up.